.github/workflows/test.yml

```diff
@@ -8,9 +8,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.11.x, 3.10.x, 3.9]
-        django: [42, 32]
+        python-version: [3.13.x, 3.12.x, 3.11.x, 3.10.x, 3.9]
+        django: [52, 42, 32]
```

Comment:
we can remove django 3.2 as it's no longer supported upstream

Reply:
@yakky Done